armbian/base/customize-armbian-rockpro64.sh

```diff
@@ -420,7 +420,8 @@ fi
 if [[ "$BASE_OVERLAYROOT" == "true" ]]; then
   # allow missing files if build-ondevice (e.g. not Armbian distro)
   if [[ -f /etc/default/armbian-ramlog ]] || [[ "${BASE_BUILDMODE}" != "ondevice" ]]; then
```

Comment:
This line already contains the check if the file to `sed` exists, which is only a requirement if the script is run on an existing distro with `make build-ondevice`. If the file does not existes when compiling Armbian from scratch, the script automatically fails due to `set -e` in the beginning.